src/jit/liveness.cpp

```diff
@@ -1626,7 +1626,7 @@ void Compiler::fgComputeLifeCall(VARSET_TP& life, GenTreeCall* call)
 //    `true` if the local var node corresponds to a dead store; `false`
 //    otherwise.
 //
-bool Compiler::fgComputeLifeLocal(VARSET_TP& life, VARSET_TP& keepAliveVars, GenTree* lclVarNode, GenTree* node)
+bool Compiler::fgComputeLifeLocal(VARSET_TP& life, VARSET_VALARG_TP keepAliveVars, GenTree* lclVarNode, GenTree* node)
 {
```

Comment:
You converted this to a value type, not a reference type. Do you want it copied? That doesn't seem right.